docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+From ocaml/opam2:ubuntu-lts
+
+RUN sudo apt-get update; sudo apt-get -y install build-essential libgmp-dev z3 m4 pkg-config zlib1g-dev device-tree-compiler; sudo apt-get -y clean
+
+RUN opam switch create ocaml-base-compiler.4.06.1
+RUN opam config env
```

Comment:
is this line for debugging purposes?

Reply:
Yes, it could be removed. I was working through Nikhil's installation slides for the tutorial that he is doing next week at the Summit and some of those steps are still in the Dockerfile.